src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -178,19 +186,24 @@ public JObject Header
         /// Gets the 'value' of the 'JWT ID' claim { jti, ''value' }.
         /// </summary>
         /// <remarks>If the 'JWT ID' claim is not found, an empty string is returned.</remarks>
-        public override string Id => Payload.Value<string>(JwtRegisteredClaimNames.Jti) ?? String.Empty;
+        public override string Id => Payload?.Value<string>(JwtRegisteredClaimNames.Jti) ?? String.Empty;
+
+        /// <summary>
+        /// Gets the <see cref="JsonWebToken"/> associated with this instance.
+        /// </summary>
+        public JsonWebToken InnerToken { get; internal set; }
 
         /// <summary>
         /// Gets the 'value' of the 'iat' claim { iat, 'value' } converted to a <see cref="DateTime"/> assuming 'value' is seconds since UnixEpoch (UTC 1970-01-01T0:0:0Z).
         /// </summary>
         /// <remarks>If the 'exp' claim is not found, then <see cref="DateTime.MinValue"/> is returned.</remarks>
-        public DateTime IssuedAt => GetDateTime(JwtRegisteredClaimNames.Iat);
+        public DateTime IssuedAt => Payload == null ? DateTime.MinValue : GetDateTime(JwtRegisteredClaimNames.Iat);
 
         /// <summary>
         /// Gets the 'value' of the 'issuer' claim { iss, 'value' }.
         /// </summary>
         /// <remarks>If the 'issuer' claim is not found, an empty string is returned.</remarks>   
```

Comment:
This is the 'iss' claim, not 'issuer' claim.

Reply:
I fixed this, as well as other occurrences of the same thing.